pep-0649.rst

```diff
@@ -556,8 +583,9 @@ code should prefer to interact with it only via the
 
 The callable stored in ``__annotate__`` must accept a
 single required positional argument called ``format``,
-which will always be an ``int``.  It must either return
-a dict (or subclass of dict) or raise
+which will always be an ``int`` or an object interchangable
```

Comment:
I rely on Google for checking my spelling, and Google says this has an e in it:
```suggestion
which will always be an ``int`` or an object interchangeable
```

Reply:
I thought the two spellings were interch... anyway I think we're removing this sentence fragment.